packages/openapi3/src/schema-emitter.ts

```diff
@@ -653,6 +660,180 @@ export class OpenAPI3SchemaEmitter extends TypeEmitter<
     }
   }
 
+  #attachXmlObjectForScalarOrModel(
```

Comment:
could we move the xml logic in its own file for clarity and would help with the dynamic loading of the lkibrary as well